host_vars/compute01.yml.example

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Rename to hv01 to align with readme / sample inventory file

Reply:
renamed.